sdk/storage/Azure.Storage.Blobs/tests/ContainerClientTests.cs

```diff
@@ -678,7 +714,7 @@ public async Task ExistsAsync_Error()
             // Act
             await TestHelper.AssertExpectedExceptionAsync<RequestFailedException>(
                 unauthorizedContainerClient.ExistsAsync(),
-                e => Assert.AreEqual(BlobErrorCode.ResourceNotFound.ToString(), e.ErrorCode.Split('\n')[0]));
+                e => Assert.AreEqual(BlobErrorCode.NoAuthenticationInformation.ToString(), e.ErrorCode.Split('\n')[0]));
```

Comment:
Is this because of a service change that's tied to the version?  We might need to wrap this in a version check.

Reply:
fixed.